src/lambdas/processDataRequest/initiateDataTransfer.ts

```diff
@@ -31,17 +31,15 @@ export const initiateDataTransfer = async (
   const copyFromAuditToAnalysisBucketRequired =
     bucketData.standardTierLocationsToCopy.length > 0
 
-  logger.info('storing new data request record')
   await addNewDataRequestRecord(dataRequestParams, glacierRestoreRequired)
-
+  logger.info('Added data request to query request database')
   if (!glacierRestoreRequired && !copyFromAuditToAnalysisBucketRequired) {
     logger.info(interpolateTemplate('dataAvailableQueuingQuery', loggingCopy))
     await sendInitiateAthenaQueryMessage(dataRequestParams.zendeskId)
     return
   }
 
   if (glacierRestoreRequired) {
-    logger.info(interpolateTemplate('foundGlacierLocations', loggingCopy))
```

Comment:
In this case, I think knowing that the Glacier Restore is going to happen is useful. Seeing as if it starts, we won't know it completes for a loooong time.

Reply:
Actually, I can see this is logged in the function itself!